tests/performance/pom.xml

```diff
@@ -54,10 +106,15 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.databene</groupId>
-            <artifactId>contiperf</artifactId>
-            <version>2.2.0</version>
-            <scope>test</scope>
-        </dependency>    
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.17.3</version>
```

Comment:
Why? Anything wrong with putting benchmarks in src/main and tests that assert benchmarks in src/test?